### PR TITLE
doc: translate  /about pages

### PR DESCRIPTION
**Pull request by h1z3y3**
Add Chinese translation of /about

### CODE REVIEW

**Comment by h1z3y3** [comment]
@myitcv 
When you review the translations. Please pay more attention on following that I'm not sure about.

* `Schema`: `规范` . At page start, I give [more information](https://github.com/cue-lang/cuelang.org/pull/238/files#diff-66343d5c39825119a0412773e89a61821b498fa6f63c725dbe8a37bc25590ee9R7) about our discuss.  
* `Push, not pull, constraints`: `主动而不是被动使用约束`. I think of `push` as `initiative` and `pull` as `passive`.

**Comment by myitcv** [comment]
> When you review the translations. Please pay more attention on following that I'm not sure about.

Will do!

Just back from holiday so catching up on things.

File: content/zh/docs/about/_index.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
++++
+title = "关于"
+weight = 100
+description = "CUE是如何诞生的，以及它的原理是什么"
++++
+
+> 译者注：原文中使用的 `Schema`，虽然直译应为 `模式`，但是个人认为此翻译并不是特别好理解。
+> 于是与 CUE 的维护者之一 [@myitcv](https://github.com/myitcv) 进行了 [讨论](https://cuelang.slack.com/archives/C032M6T7WAG/p1645762537397239) ，
+> 他认为 `specification` 或 `struct` 会更接近，因此下文将 `Schema` 翻译为 `规范`。
```
Comment (FogDong) [inline content/zh/docs/about/_index.md:9]: 这块修改一下，将后面的改为 `Schema`？

File: content/zh/docs/about/_index.md [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+- 配置：值可以从不同来源进行组合，而不需要其中一个必须引用另一个。
+
+值 排序的过程也能对整个配置进行集合包含分析。
+当大多数校验系统只能检查一个具体的值是否匹配定义时，CUE 能校验某个 规范 的实例也是另一个 规范 的实例（是否向后兼容？）
+或计算一个新的 规范，此 规范 能表示与其他两种 规范 相匹配的所有实例。
+
+## 历史
+
+虽然 CUE 是一门特别不同的语言，但它的根源是 GCL（Google Config Language），到目前 GCL 是 Google 主要使用的配置语言。
+它最初被设计用于配置 Kubernetes 的前身 Borg。实际上，最开始想的是使用 CUE 中使用的图归一化实现 GCL。
```
Comment (FogDong) [inline content/zh/docs/about/_index.md:41]:
```suggestion
它最初是为配置 Kubernetes 的前身 Borg 而设计的。实际上，最开始想的是在 GCL 中使用 CUE 中的图归一化。
```

File: content/zh/docs/about/_index.md [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+
+**大规模**
+CUE 就是为大规模配置而生，它的导入模型（import model）包含了大规模开发的最佳实践，而且对自动化进行了优化。
+其中一个关键点是高级工具，CUE 的操作所依赖的数学模型（mathematical model）允许实现自动化，这对于大多数其他方法来说是无法实现的。
+CUE 的 `trim` 命令就是其中一个例子。
+
+### 工具 
+
+自动化是关键。
+o
```
Comment (FogDong) [inline content/zh/docs/about/_index.md:216]: What's this?

File: content/zh/docs/about/_index.md [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+
+虽然 CUE 是一门特别不同的语言，但它的根源是 GCL（Google Config Language），到目前 GCL 是 Google 主要使用的配置语言。
+它最初是为配置 Kubernetes 的前身 Borg 而设计的。实际上，最开始想的是在 GCL 中使用 CUE 中的图归一化。
+GCL 的一位作者对实现这类系统有丰富的经验，也体验到了能够通过计算和推理类型以创建强大工具的好处。
+
+CUE 所基于的图归一化模型是当时在计算机语言学中广泛使用的，并且曾成功管理超过 10 万行声明式定义的语法和词典。
+这些实际上是像人类语言这样不规则和复杂的非常大的配置，这些系统一个特点就是定义的 类型 或 约束条件 可以验证数据，并减少重复使用的代码块。
+总的来说，这种方法非常适合云上的配置。 
+
+然而，GCL 的早期设计追求的是更简单的东西，巧合的是它与图归一化的概念不相容。
```
Comment (FogDong) [inline content/zh/docs/about/_index.md:47]:
```suggestion
然而，GCL 的早期设计追求的是更简单的东西，巧的是这种设计与图归一化的概念不相容。这种简单的设计后来也被证明是远远不够的，但此时想要转向之前的设计已经太晚了。与图归一化相反的，GCL 采用了基于继承的覆盖模型。这种模型的复杂性使得早期的工具难以实现，也从未实现。复制其模型的 GCL 后代也是如此。
```

File: content/zh/docs/about/_index.md [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+但是它的复杂性使得早期的工具难以实现，也从未实现，复制其模型的 GCL 后代也是如此。
+
+CUE 回归使用基于约束的最初想法，并努力从使用 GCL 15 年的经验中吸取教训。
+同时，也从后代中吸取了经验教训以及完全不同的配置方法。
+
+## 理念与原则
+
+### 类型 和 值
+
+CUE 没有区分 值 和 类型，这是一个超强的概念，能让 CUE 定义非常详细的约束条件，同时也极大地简化了很多事情：
```
Comment (FogDong) [inline content/zh/docs/about/_index.md:59]: 我建议只在中英文之间加空格，其他特殊意义的值可以用 ``，
如： CUE 没有区分`值`和`类型`。

File: content/zh/docs/about/_index.md [hunk 6]
```diff
@@ -0,0 +1,235 @@
...
+CUE 中每一个字段声明定义一组节点来应用具体的约束条件。
+因为顺序是无关的，多个约束条件可以应用到同一个节点，所有的约束条件需要同时应用。
+这些约束条件可能甚至会在不同文件中。
+但是它们不会相互矛盾：
+如果一个声明一个字段值是 `5`，另一个不能使用 `6` 来覆盖它，
+但是同时声明 `>5` 和 `<10` 是有效的。
+
+这样的方法比全面继承的方法更有局限性：
+它不太可能复用已存在的配置。但另一方面，它在移除引用时更强大。
+比如，假设一组工作中的每个工作都需要使用特定的模板,
```
Comment (FogDong) [inline content/zh/docs/about/_index.md:148]: BTW，为什么每个断句都要换行？

File: content/zh/docs/about/_index.md [hunk 7]
```diff
@@ -0,0 +1,235 @@
...
+
+**小规模**
+当项目规模很小时，减少配置中的重复代码块并不是一定要做的事。
+但是即使规模很小，重复也容易出错。对于这种情况，CUE 可以定义 Schema 用于验证无类型数据文件的有效性。
+
+**中规模**
+只要想要减少重复代码块，`cue` 工具就可以帮助来自动重写配置。
+请到 [Kubernetes tutorial](/docs/tutorials/kubernetes) 的 Quick and Dirty 
+这一节查看关于使用 `import` 和 `trim` 工具的例子。
+使用这个方法，可以清理成千上万行。
```
Comment (FogDong) [inline content/zh/docs/about/_index.md:206]:
```suggestion
使用这个方法，可以清理成千上万行代码。
```

**Pull request closed without merging**